descartes_light/solvers/include/descartes_light/solvers/bgl/bgl_solver.h

```diff
@@ -65,7 +65,7 @@ class BGLSolverBase : public Solver<FloatType>
    * should start from this vertex */
   VertexDesc<FloatType> source_;
   /** @brief Predecessor map generated by the search */
-  std::map<VertexDesc<FloatType>, VertexDesc<FloatType>> predecessor_map_;
+  std::vector<VertexDesc<FloatType>> predecessor_map_;
```

Comment:
This should probably be renamed since it is no longer a map

Reply:
```suggestion
  std::vector<VertexDesc<FloatType>> predecessors_;
```